scripts/set_requester_pays.sh

```diff
@@ -1,86 +1,88 @@
 #!/bin/bash
 
-if (( $# < 2 )); then
+# check that there are only 3 arguments passed
+if (( $# < 3 )); then
+  echo "Usage: $0 [on | off] TERRA_PROJECT_ID TERRA_BUCKET_PATH"
+  echo 'The Terra bucket path can be formatted as "gs://fc-XXXXX" or "fc-XXXXX"'
+  echo "i.e you can run './set_requester_pays.sh on project_id fc-12345'"
+  echo "NOTE: this script requires you to be authed as your firecloud.org admin account."
+  exit 0
+elif (( $# > 3 )); then
   echo "Usage: $0 [on | off] TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
```

Comment:
you could merge the check for <3 and >3 to just != (or maybe -ne) 3 and then print the usage.